lib/internal/util/inspect.js

```diff
@@ -636,7 +653,18 @@ const wellKnownPrototypes = new SafeMap()
   .set(RegExpPrototype, { name: 'RegExp', constructor: RegExp })
   .set(DatePrototype, { name: 'Date', constructor: Date })
   .set(DataViewPrototype, { name: 'DataView', constructor: DataView })
+
   .set(ErrorPrototype, { name: 'Error', constructor: Error })
+  .set(AggregateErrorPrototype, { name: 'AggregateError', constructor: AggregateError })
+  .set(EvalErrorPrototype, { name: 'EvalError', constructor: EvalError })
+  .set(RangeErrorPrototype, { name: 'RangeError', constructor: RangeError })
+  .set(ReferenceErrorPrototype, { name: 'ReferenceError', constructor: ReferenceError })
+  // TODO for when Node supports SuppressedError
```

Comment:
```suggestion
  // TODO for when Node supports SuppressedError
```
```suggestion
  // TODO(erights): for when Node supports SuppressedError
```

Reply:
Hi @aduh95 ,
Does "TODO(erights):" mean that erights (me) is the one expected to do this? I do not expect to volunteer for that ;). I'm just trying to be helpful to Node maintainers.

Reply:
If I should indeed remove the others such comments (as you suggest and I've already committed), then should I also just remove this comment?